std/net/curl.d

```diff
@@ -214,21 +215,32 @@ version(unittest)
         import std.concurrency : spawn;
         import std.socket : INADDR_LOOPBACK, InternetAddress, TcpSocket;
 
+        tlsInit = true;
```

Comment:
Is this ever _not_ true?

Reply:
tlsInit stays false in all threads that did not start the TestServer.

Reply:
I thought it was something like that but couldn't tell from the patch.